src/test/scala/RegexTokenizerTest.scala

```diff
@@ -0,0 +1,182 @@
+ * with other code, such other code is not for that reason alone subject to any
+ * of the requirements of the GNU Affero GPL version 3 as long as this Program
+ * is the same Program as licensed from Suomen Kanuuna Oy without any additional
+ * modifications.
+ *
+ * Supplemented terms under GNU Affero General Public License version 3
+ * section 7
+ *
+ * Origin of the software must be attributed to Suomen Kanuuna Oy. Any modified
+ * versions must be marked as "Modified version of" The Program.
+ *
+ * Names of the licensors and authors may not be used for publicity purposes.
+ *
+ * No rights are granted for use of trade names, trademarks, or service marks
+ * which are in The Program if any.
+ *
+ * Licensee must indemnify licensors and authors for any liability that these
+ * contractual assumptions impose on licensors and authors.
+ *
+ * To the extent this program is licensed as part of the Commercial versions of
+ * Teragrep, the applicable Commercial License may apply to this file if you as
+ * a licensee so wish it.
+ */
+
+import com.teragrep.functions.dpf_03.{ByteArrayListAsStringListUDF, RegexTokenizerUDF, TokenizerUDF}
+import org.apache.spark.sql._
+import org.apache.spark.sql.catalyst.encoders.RowEncoder
+import org.apache.spark.sql.execution.streaming.MemoryStream
+import org.apache.spark.sql.streaming.{StreamingQuery, Trigger}
+import org.apache.spark.sql.types._
+
+import java.sql.Timestamp
+import java.time.{Instant, LocalDateTime, ZoneOffset}
+import scala.collection.mutable
+import scala.collection.mutable.ArrayBuffer
+
+class RegexTokenizerTest {
+  val exampleString: String = "NetScreen row=[Root]system-notification-00257" +
+    "(traffic\uD83D\uDE41 start_time=\"2022-09-02 10:13:40\"" +
+    " duration=0 policy_id=320000 service=tcp/port:8151 proto=6" +
+    " src zone=Null dst zone=Null action=Deny sent=0 rcvd=40" +
+    " src=127.127.127.127 dst=127.0.0.1 src_port=52362" +
+    " dst_port=8151 session_id=0 reason=Traffic Denied"
+
+  val amount: Long = 10
+
+  val testSchema: StructType = new StructType(
+    Array[StructField]
+      (StructField("_time", TimestampType, nullable = false, new MetadataBuilder().build),
+        StructField("_raw", StringType, nullable = false, new MetadataBuilder().build),
+        StructField("index", StringType, nullable = false, new MetadataBuilder().build),
+        StructField("sourcetype", StringType, nullable = false, new MetadataBuilder().build),
+        StructField("host", StringType, nullable = false, new MetadataBuilder().build),
+        StructField("source", StringType, nullable = false, new MetadataBuilder().build),
+        StructField("partition", StringType, nullable = false, new MetadataBuilder().build),
+        StructField("offset", LongType, nullable = false, new MetadataBuilder().build),
+        StructField("estimate(tokens)", LongType, nullable = false, new MetadataBuilder().build)
+      )
+  )
+
+  @org.junit.jupiter.api.Test
```

Comment:
Looks like junit4 style, it is enabled along with junit5 in pom

Reply:
all tests use junit5 now removed junit4 from pom